financial-connections/src/main/AndroidManifest.xml

```diff
@@ -67,18 +67,4 @@
             android:multiprocess="true" />
     </application>
 
-    <!--
-    See https://developer.android.com/training/package-visibility/declaring for more details.
-    -->
-    <queries>
-        <!--
-        Added to check the default browser that will host the AuthFlow.
-        -->
-        <intent>
-            <action android:name="android.intent.action.VIEW" />
```

Comment:
is this change related to customtab?

Reply:
it is:

- we need the http permission to query the browsers available on the device -> https://github.com/stripe/stripe-android/pull/7081/files#diff-0967448bd0d3c93d58931a149ec7d1884ae936cfffd5a5eddb3c67acf49b6b70R25-R29
- This is needed to fall back to a non-default browser if the default browser does not support Custom tabs. See: https://github.com/GoogleChrome/android-browser-helper/blob/4f43bbfc802b963d2e924577e460a216a01d9700/demos/custom-tabs-example-app/src/main/java/org/chromium/customtabsdemos/CustomTabsHelper.java#L75-L85

Reply:
Also this is the different outputs we'd get depending on the permisions: <img width="1450" alt="image" src="https://github.com/stripe/stripe-android/assets/99293320/1083bf51-e4c1-4c27-8cba-5bde68cdc91f">